doc/getting-started.md

```diff
@@ -0,0 +1,584 @@
+# Getting Started
+
+This section will serve as an introductory Mio tutorial. It assumes that
+you have familiarity with the [Rust](http://www.rust-lang.org/)
+programming language and the [Cargo](https://crates.io) tool. It will
+start from generating a new [Rust](http://www.rust-lang.org/) project
+using [Cargo](https://crates.io) up to writing a simple TCP echo server
+and client.
+
+Of course, you will need Rust installed. If you haven't already, get it
+here: [rust-lang.org](https://www.rust-lang.org).
+
+The complete echo server can be found
+[here](../examples/ping_pong/src/server.rs). A client example is
+[here](../examples/ping_pong/src/client.rs).
+
+> **Note:** As of the time of writing, Mio does not support Windows
+> (though support is currently in progress).
```

Comment:
This disagrees with the [README](https://github.com/carllerche/mio/commit/168fcb5938897eadc9f57e5bc52ae148108e1778#diff-04c6e90faac2675aa89e2176d2eec7d8R59)?